system/ee/language/english/fieldtypes_lang.php

```diff
@@ -49,7 +49,7 @@
 
     'rel_ft_limit' => 'Maximum entries',
```

Comment:
I think we need to update this label to reduce confusion. What about: "Number of available entries displayed"

Reply:
I'd keep it short (however I don't quite like the existing option name either)